velox/type/Type.cpp

```diff
@@ -1011,6 +1011,22 @@ std::string DateType::toString(int32_t days) const {
   return Timestamp::tmToString(tmValue, 0, options);
 }
 
+std::string DateType::toISO8601String(int32_t days) const {
```

Comment:
Is this a copy-paste from DateType::toString? We have discussed that we should avoid copy-pasting by calling this new function from toString.

Reply:
Fixed.